src/modules/auth/login.strategy.ts

```diff
@@ -2,29 +2,42 @@ import { LoginStrategy } from 'passport-did-auth';
 import { Inject, Injectable } from '@nestjs/common';
 import { PassportStrategy } from '@nestjs/passport';
 import { ConfigService } from '@nestjs/config';
+import { LoginStrategyOptions } from 'passport-did-auth/dist/lib/LoginStrategy';
```

Comment:
@whitneypurdum I don't think if it would be a good idea to update `passport-did-auth` now, given the long login issue and also there are some changes in the pipeline reg the `iam-client-examples` update. So the behaviours and signature may change. @jrhender what do you think about this?

Reply:
https://www.npmjs.com/package/passport-did-auth

Reply:
I just reverted it back to the original version!